legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/defaultExtension.pure

```diff
@@ -620,3 +625,108 @@ function meta::relational::functions::sqlQueryToString::default::escapeLikeExprD
       ->replace('_', '\\_')
       ->replace('%', '\\%');
 }
+
+function meta::relational::functions::sqlQueryToString::default::sqlStringToUpdateNode(sql: String[*]): meta::relational::executionPlan::UpdateSqlExecutionNode[1]
```

Comment:
executionPlan/executionNode related fns should not be defined in sqlQueryToString. Move them to other relevant files.